eng/pipelines/runtime.yml

```diff
@@ -50,6 +50,10 @@ pr:
 
 variables:
   - template: /eng/pipelines/common/variables.yml
+  - name: Codeql.Enabled
```

Comment:
We only want to turn this on in runtime-official.yml.

Reply:
Done, migrated to runtime-official.yml